### DISCUSSION TITLE
Changing paths, migrating comments

### DISCUSSION THREAD

**Discussion by stevygee** [walinejs/waline#1463] [Q&A] [answered]
Hi,
I'm thinking of using this, but my page paths might change in the future. Is there a way to migrate comments from one page (path) to another?

**Comment by lizheming** [comment, marked as answer]
`path` parameter just a unique id to connect comment data and your article page. You can use any string text for it, but Waline use path of the article page as default value. If you'll change your blog path rule in the future, I recommend you use a changeless text instead of path. 

If you still use path, there has two ways to satisfy your demand.
1. You can change comment data path field value directly in the LeanCloud dashboard, and also for other storage service.
    ![image](https://user-images.githubusercontent.com/424491/193861254-ffa58998-cb22-4d01-b3e4-5ed7917c50be.png)
    ![image](https://user-images.githubusercontent.com/424491/193861354-19eb2de0-90f6-41c3-93ba-7d62ec96365a.png)
2. use a map record your new path and old path, then change `path` parameter when you init Waline.
